mobify/velocity
# Creating custom velocity build for integration with Zepto

Removed generic shim code in favor of Zepto specific shim.

Status: **Opened for visibility**

Reviewers: @kpeatt @tedtate
## Changes
- Removed shim functions that were duplicated in Zepto
- Removed CommonJS definition in UMD wrapper
- Updated UMD wrapper to use correct dependencies
## How to test-drive this PR
- `bower install` from this branch into a project that uses Velocity
- make sure it works
